src/main/java/dci/j24e01/OurShop/controllers/ProductController.java

```diff
@@ -23,11 +24,13 @@ public class ProductController {
     ProductDAO productDAO;
 
     @GetMapping("/products")
-    public String list(Model model) {
-        List<Product> products = productDAO.getProducts();
-        model.addAttribute("products", products);
+    public ModelAndView list() {
```

Comment:
This is nice.
And it's important to explore nice things...

But it;s not consistent with the rest of the Controllers...